tasks/__init__.py

```diff
@@ -504,6 +519,10 @@ def test_travis_else():
     karma(single=True, browsers='PhantomJS')
 
 @task
+def test_travis_varnish():
```

Comment:
Yeah, I was just trying to match the existing standard. It looks like test_travis_else and test_travis_osf end up calling other methods and just acting as a wrapper. test_travis_varnish only has one call but keeping test_varnish would allow people to call it from their development machines and not have to think about what the differences between test_travis_varnish and test_varnish are.